docs/src/components/homepage/styles.module.css

```
@@ -45,73 +49,121 @@
   padding: 0 2rem;
 }
 
-.hero {
+@keyframes orbDrift {
```

Stylelint (17.12.0)

[error] Expected keyframe name "orbDrift" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)